ldes-server-infra-mongo/mongo-repository/src/test/java/be/vlaanderen/informatievlaanderen/ldes/server/infra/mongo/membersequence/service/SequenceGeneratorServiceTest.java

```diff
@@ -0,0 +1,42 @@
+package be.vlaanderen.informatievlaanderen.ldes.server.infra.mongo.membersequence.service;
+
+import be.vlaanderen.informatievlaanderen.ldes.server.infra.mongo.membersequence.entity.MemberSequenceEntity;
+import org.junit.jupiter.api.Test;
+import org.springframework.data.mongodb.core.MongoOperations;
+import org.springframework.data.mongodb.core.query.Update;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import static org.springframework.data.mongodb.core.query.Criteria.where;
+import static org.springframework.data.mongodb.core.query.Query.query;
+
+class SequenceGeneratorServiceTest {
```

Comment:
maybe we can use a test to make sure that two collections have separate sequences?

Reply:
We should have an integration test for this, but I'm noticed that the Mongo integration tests are not working:
https://github.com/Informatievlaanderen/VSDS-LDESServer4J/issues/510

I updated the issue with a request to also add integration tests for this.